cmd/meta/server.go

```diff
@@ -38,20 +39,27 @@ type Server struct {
 	store  *meta.Store
 }
 
-func NewServer(store *meta.Store, musicbrainzDB *sql.DB) (*Server, error) {
+func NewServer(store *meta.Store, DB *sql.DB) (*Server, error) {
 	srv := &Server{
 		router: httprouter.New(),
 		store:  store,
 	}
 	srv.router.GET("/object/:cid", srv.HandleGetObject)
 	srv.router.POST("/import/xml", srv.HandleImportXML)
-	if musicbrainzDB != nil {
-		api, err := musicbrainz.NewAPI(musicbrainzDB, store)
+	if DB != nil {
+		musicbrainzApi, err := musicbrainz.NewAPI(DB, store)
 		if err != nil {
 			return nil, err
 		}
-		srv.router.Handler("GET", "/musicbrainz/*path", http.StripPrefix("/musicbrainz", api))
-		srv.router.Handler("POST", "/musicbrainz/*path", http.StripPrefix("/musicbrainz", api))
+		srv.router.Handler("GET", "/musicbrainz/*path", http.StripPrefix("/musicbrainz", musicbrainzApi))
+		srv.router.Handler("POST", "/musicbrainz/*path", http.StripPrefix("/musicbrainz", musicbrainzApi))
+
+		cwrApi, err := cwr.NewAPI(DB, store)
```

Comment:
This code is still using the same database for both MusicBrainz and CWR, they should be separate.